modules/javafx.graphics/src/main/java/com/sun/javafx/text/PrismTextLayout.java

```diff
@@ -648,6 +649,18 @@ public Shape getShape(int type, TextSpan filter) {
         return outline;
     }
 
+    @Override
+    public boolean setTabSize(int spaces) {
+        if (spaces < 1)
+            spaces = 1;
```

Comment:
Please surround the statement with curly braces (our coding style is to always surround the target of a conditional in curly braces unless it is on the same line).